packages/@webex/media-helpers/src/index.ts

```diff
@@ -14,8 +14,10 @@ export {
   LocalCameraStream,
   createMicrophoneStream,
   createCameraStream,
+  createCameraAndMicrophoneStreams,
```

Comment:
nitpick: feels like our naming between user media and display media is not very consistent. `createCameraAndMicrophoneStreams` is not consistent with `createDisplayStreamWithAudio` - one has "and", the other "with" and one expicitly says what streams are created the other just says "withAudio". Is there a plan to maybe deprecate `createDisplayStreamWithAudio` now that we have the new ` createDisplayMedia()`? If we have now `createDisplayMedia()` then maybe we should have `createUserMedia()` for mic and camera?

Reply:
@bbaldino had originally suggested deprecating `createDisplayStream` and `createDisplayStreamWithAudio` but we ultimately decided to keep them as "simple" versions of the API for devs who are not interested in using any of the advanced options. In terms of why the original APIs were named differently, honestly I think it's mostly because `createDisplayStreamWithAudio` came first, and it felt weird to name the other API `createCameraStreamWithAudio`. But in my mind there is a slight difference that may justify the different name... you can create camera and microphone streams separately (and we have APIs for it) but it is not possible to create a `SystemAudioStream` without a `DisplayStream`.

Reply:
Also I'm not opposed to having a `createUserMedia` API but there aren't any additional options for `getUserMedia` like `getDisplayMedia` does to justify having another API, so we'd basically only be doing it for consistency.

Reply:
@marcin-bazyl it's a fair point on the naming.  For what it's worth the display stuff is genuinely different: for example you can't get only an audio stream, so there's not really any concept of "just" a system audio stream, so I think that motivated some of those naming differences: you can only get a display stream or a display stream _with_ audio as well.  As opposed to main media where you can get audio, video or audio _and_ video.  But maybe they can be better/more consistent.

Reply:
ok, @brycetham @bbaldino fair enough, let's keep it different for main media and screenshare then.